cla_public/static-src/javascripts/modules/form-errors.js

```diff
@@ -175,6 +185,8 @@
 
         var label = $('#field-label-' + fieldName);
 
+        if (label.parents(".cla-currency-by-frequency").length) label = label.parents(".cla-currency-by-frequency").children("legend");
```

Comment:
Can this be split out into multiple lines to make readability easier i.e. 
`if (condition) {
...
}`